src/plugins/event_annotation/kibana.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "eventAnnotation",
+  "version": "kibana",
+  "server": true,
+  "ui": true,
+  "extraPublicDirs": [
+    "common"
+  ],
+  "requiredPlugins": [
```

Comment:
I think that it will be good to also add the description here. I think it is connected with our documentation.